src/App.tsx

```diff
@@ -1,109 +1,38 @@
-import React from 'react';
-
 import './App.scss';
+import React from 'react';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { User } from './components/types/User';
+import { Comment } from './components/types/Comment';
+import { PostList } from './components/PostList/PostList';
+import { Post } from './components/types/Posts';
+
+function getUserById(users: User[], postId: number): User | null {
+  return users.find(user => (
+    user.id === postId
+  )) || null;
+}
+
+function getCommentsByPostId(comments: Comment[], postId: number) {
```

Comment:
You can just use `commentsFromServer` here, you don't need to pass it as an argument
```suggestion
function getCommentsByPostId( postId: number) {
```

Reply:
same with users